homeassistant/components/evohome/coordinator.py

```diff
@@ -107,7 +106,7 @@ async def call_client_api(
             return None
 
         if update_state:  # wait a moment for system to quiesce before updating state
-            async_call_later(self.hass, 1, self._update_v2_api_state)
+            await self.hass.data[DOMAIN]["coordinator"].async_request_refresh()
```

Comment:
Using `hass.data` is a bit unsatisfactory, but I am about to implement config flow for **evohome**, and I think doing it this way strikes a balance between a small/simple change and a more complex fix.